File: Rest-Api-V3/market-data-public/tickers/tickers.js
const axios = require('axios');

async function fetchTickerData(marketCode) {
  const url = `https://api.ox.fun/v3/tickers?marketCode=${marketCode}`;

  try {
    const response = await axios.get(url);
    console.log('Ticker Data:', response.data);
  } catch (error) {
    console.error('Error fetching ticker data:', error.response ? error.response.data : error.message);
  }
}

// Example usage
fetchTickerData('BTC-USD-SWAP-LIN'); 

File: Rest-Api-V3/market-data-public/tickers/tickers.py
import requests

def fetch_ticker_data(market_code):
    url = f'https://api.ox.fun/v3/tickers?marketCode={market_code}'

    try:
        response = requests.get(url)
        response.raise_for_status()  # Raise an error for bad responses
        print('Ticker Data:', response.json())
    except requests.exceptions.RequestException as error:
        print('Error fetching ticker data:', error)

# Example usage
fetch_ticker_data('BTC-USD-SWAP-LIN') 